app/models/message.rb

```diff
@@ -6,20 +6,21 @@
 #  content       :text
 #  category      :string(255)
 #  sent_at       :datetime
-#  assessment_id :integer
+#  tool_id       :integer
 #  created_at    :datetime
 #  updated_at    :datetime
 #  mandrill_id   :string(255)
 #  mandrill_html :text
+#  tool_type     :string
 #
 
 class Message < ActiveRecord::Base
-	belongs_to :assessment
+	belongs_to :tool, polymorphic: true
 	MAX_LENGTH_TEASER = 220
 
 	def teaser
 	  return content unless content
     return content if content.length <= MAX_LENGTH_TEASER
-    return "#{content[0..MAX_LENGTH_TEASER-1]}..."
+    "#{content[0..MAX_LENGTH_TEASER-1]}..."
```

Comment:
This is the way the original code behaved.  It would truncate what existed in the database down to 220 characters, then tack on an additional 3.